tests/test_xsd_any.py

```diff
@@ -218,6 +218,63 @@ def test_element_any_type():
     assert item.something == 'bar'
 
 
+def test_array_of_element_any_type():
+    node = etree.fromstring("""
+        <?xml version="1.0"?>
+        <schema xmlns="http://www.w3.org/2001/XMLSchema"
+                xmlns:tns="http://tests.python-zeep.org/"
+                targetNamespace="http://tests.python-zeep.org/"
+                elementFormDefault="qualified">
+          <element name="container">
+            <complexType>
+              <sequence>
+                <element name="properties" type="tns:array_of_property"/>
+              </sequence>
+            </complexType>
+          </element>
+          <complexType name="array_of_property">
+            <sequence>
+              <element maxOccurs="unbounded" minOccurs="0" name="property" type="tns:property"/>
+            </sequence>
+          </complexType>
+          <complexType name="property">
+            <sequence>
+              <element minOccurs="0" name="value" type="anyType"/>
+            </sequence>
+            <attribute name="name" type="string" use="required"/>
+          </complexType>
+        </schema>
+    """.strip())
+    schema = xsd.Schema(node)
+
+    container_element = schema.get_element('{http://tests.python-zeep.org/}container')
+    array_of_property_type = schema.get_type('{http://tests.python-zeep.org/}array_of_property')
+    property_type = schema.get_type('{http://tests.python-zeep.org/}property')
+
+    property1 = property_type(name='a name', value=xsd.AnyObject(xsd.String(), 'a value'))
+    array_of_property = array_of_property_type(property=[property1])
+    container = container_element(properties=array_of_property)
+
+    node = etree.Element('document')
+    container_element.render(node, container)
```

Comment:
if you add etree.cleanup_namespaces(node) below this line then you will trigger the bug